core/templates/dev/head/components/summary-tile/story-summary-tile.directive.ts

```diff
@@ -36,7 +36,7 @@ angular.module('oppia').directive('storySummaryTile', [
           var ctrl = this;
           ctrl.getStoryLink = function() {
             return UrlInterpolationService.getStoryUrl(
-              '/story', ctrl.getStoryId());
+              '/' + ctrl.getStoryId());
```

Comment:
I am fixing this change in the subtopic page PR that I'll be creating shortly :).

Reply:
Oh okay! I was going to create a PR for the pretest fix and then noticed this so I did that as well. If this was the fix you intended to make then I can submit that in the PR, otherwise I will revert this and wait for your PR to go in!

Reply:
My PR should be merged by tomorrow at the latest. If that's fine, then you can revert the changes here and wait for mine to go in.

Reply:
Got it , reverted the changes